### PR TITLE
feat: added login page

### PR DESCRIPTION
**Pull request by amirnathani9**
## changelog
- added login page

## screenshots
image

## preview link
https://deploy-preview-7--watchesworld.netlify.app/pages/login.html

### CODE REVIEW

File: style/login.css [hunk 1]
```diff
@@ -0,0 +1,26 @@
+.login-section{
+    height: 91vh;
+}
+.login-main-container{
+    padding: 2rem;
+    box-shadow: 5px 5px 10px rgba(2, 129, 149, 0.3);
+}
+.btn{
+    margin: 1rem 0rem 1rem 0rem;
```
Comment (yashpatel1797) [inline style/login.css]: margin: 1rem 0rem; will work also.

**Review by yashpatel1797 — APPROVED** [review]
looks good to merge.

File: pages/login.html [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+                <a href="" class="border-radius-2"><i class="fas fa-shopping-cart"></i>Cart</a>
+            </li>
+        </ul>
+    </div>
+</nav>
+<!-- login section -->
+<section class="login-section section-center flex justify-center items-center">
+    <div class="login-main-container flex flex-col items-center border-radius-2">
+        <h1 class="section-heading font-bold border-radius-1 p-2 my-4">Login</h1>
+        <form class="">
```
Comment (sharmayatendra) [inline pages/login.html]: class=""?

**Review by sharmayatendra — APPROVED** [review]
good work can look upon suggested changes

**Comment by aurobindoGupta** [comment]
nice work

File: pages/login.html (hunk 2 above)
Comment (amirnathani9) [reply pages/login.html]: yaa removed

File: style/login.css (hunk 1 above)
Comment (amirnathani9) [reply style/login.css]: okk. Done with the change

**Pull request merged**